libraries/natives.py

```diff
@@ -39,41 +39,40 @@ def display_help(self):
         ?:          Display this message
         """)
 
-#todo add force or attach
-    def dump(self,session,lib,free=False,base_address=None,size=None,package_name=''):
+    # todo add force or attach
+    def dump(self, session, lib, free=False, base_address=None, size=None, package_name=''):
         try:
             path = '.'
-            script = session.create_script(open(os.path.dirname(__file__)+"/js/memops.js").read())
+            script = session.create_script(open(os.path.dirname(__file__) + "/js/memops.js").read())
             script.load()
             api = script.exports
             if not free:
                 dump_area = api.moduleaddress(lib)
                 for area in dump_area:
-                    bs = api.memorydump(area["addr"],area["size"])
+                    bs = api.memorydump(area["addr"], area["size"])
             else:
-                bs = api.memorydump(base_address,size)  
-            
+                bs = api.memorydump(base_address, size)
+
             if package_name == '':
                 print("package name empty")
-                filepath='./dump'+os.path.sep
+                filepath = './dump' + os.path.sep
             else:
-                filepath='./dump'+os.path.sep+package_name+os.path.sep
-            
-            if not os.path.exists(filepath):
-                os.makedirs(filepath)  
+                filepath = './dump' + os.path.sep + package_name + os.path.sep
 
+            if not os.path.exists(filepath):
+                os.makedirs(filepath)
 
-            with open(filepath+lib + ".dat", 'wb') as out:
+            with open(filepath + lib + ".dat", 'wb') as out:
                 out.write(bs)
-            click.secho('[+] dump saved to {}'.format(filepath+lib + ".dat"), fg='green')
+            click.secho(f"[+] dump saved to {filepath + lib + '.dat'}", fg='green')
 
         except Exception as e:
-            click.secho("[Except] - {}:".format(e), bg='red')
+            click.secho(f"[Except] - {e}:", bg='red')
 
-    def form_bytes(self,bytes):
-        return '[%s]' % ','.join(["0x%02x" % int(x, 16) for x in bytes.split(' ')])
+    def form_bytes(self, bytes):
+        return f'[%s]' % ','.join(["0x%02x" % int(x, 16) for x in bytes.split(' ')])
```

Comment:
@jxdv f here doesn't seem to format anything

Reply:
Removed